css/index.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["../src/scss/root/_reset.scss","../src/scss/root/_a11y.scss","../src/scss/root/_index.scss"],"names":[],"mappings":"AAAA;AAAA;AAAA;AAAA;AAKA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;EAaC;EACA;EACA;EACA;EACA;EACA;;;AAED;AACA;AAAA;EAEC;;;AAED;EACC;;;AAED;EACC;;;AAED;AAAA;EAEC;EACA;;;AAED;EACC;EACA;;;AAGD;EACC;EACA;EACA;;;ACjDD;EACC;EACA;EACA;EACA;EACA;EACA;EACA;;;ACoDD;AAgDA","file":"index.css"}
```

Comment:
This goes with the `index.css` file - did you mean to commit it?

Reply:
removing